src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisDateTime.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.ndmis.performance
+
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+
+// Agreement: the v2 NDMIS report uses this format, v3 switches to proper ISO8601
+// Deadline: stop producing v2 format after 13 December 2021
+data class NdmisDateTime(val dateTime: OffsetDateTime) {
+  override fun toString(): String {
```

Comment:
This all hinges on the serialiser using `toString` -- is that the case?

Reply:
yes  - you can test this by adding to the `CsvLineAggregatorTest`